tests/dialects/test_duckdb.py

```diff
@@ -352,6 +352,7 @@ def test_duckdb(self):
     def test_time(self):
         self.validate_identity("SELECT CURRENT_DATE")
         self.validate_identity("SELECT CURRENT_TIMESTAMP")
+        self.validate_identity("VAR_POP(a)")
```

Comment:
Can we move this outside of `test_time`? Let's also add a test to demonstrate transpilation works as expected (i.e. `VARIANCE_POP -> VAR_POP`).